src/Components/Users/ManageUsers.tsx

```diff
@@ -639,6 +639,15 @@ function UserFacilities(props: { user: any }) {
       body: { home_facility: facility.id.toString() },
     });
     if (res && res.status === 200) user.home_facility_object = facility;
+    if (res?.status !== 200) {
+      Notification.Error({
```

Comment:
```suggestion
    if (!res?.ok) {
      user.home_facility_object = facility;
      Notification.Error({
```

Reply:
Sure, here's a revised version of your comment for a GitHub pull request:

Hi @rithviknishad,

I just wanted to confirm the proposed changes. It seems like user.home_facility_object = facility; should be moved to the else statement. If this adjustment aligns with our intentions, I can proceed with pushing the code.

Let me know if there are any concerns.

![image](https://github.com/coronasafe/care_fe/assets/104725768/20711b81-0875-41a4-8f3f-3f7799bde2cc)

Reply:
oh yes, you are right. my bad